src/openvas.c

```diff
@@ -45,6 +45,7 @@
 #include <fcntl.h>  /* for open() */
 #include <gcrypt.h> /* for gcry_control */
 #include <glib.h>
+#include <gnutls/gnutls.h> /* for gnutls_check_version */
```

Comment:
I guess this include is used for gnutls_global_set_log_function and gnutls_global_set_log_level because gnutls_check_version isn't added / used here.